src/script/client/ClientEntity.ts

```diff
@@ -24,11 +24,19 @@ import {splitFingerprint} from 'Util/StringUtil';
 
 import {ClientMapper} from './ClientMapper';
 
+import {isObject} from '../guards/common';
 import {ClientRecord} from '../storage';
 
-export enum MLSPublicKeys {
-  ED25519 = 'ed25519',
-}
+export const MLSPublicKeys = {
+  ed25519: 'ED25519',
+  ed448: 'ED448',
+  ecdsa_secp521r1_sha512: 'EDCSA_SECP521R1_SHA512',
+  ecdsa_secp384r1_sha384: 'EDCSA_SECP384R1_SHA384',
+  ecdsa_secp256r1_sha256: 'EDCSA_SECP256R1_SHA256',
+} as const;
```

Comment:
We don't align with iOS for those.
FYI we're using those strings in the conversation details with `Cyphersuite[<number>]` (enum from Core-Crypto)